pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
Do we want to create a separate "dev" tag?

On Apr 27, 2016, at 10:58 PM, Ben Parees <notifications@github.com> wrote:

In pkg/build/api/v1/types.go
<https://github.com/openshift/origin/pull/7949#discussion_r61367900>:

> @@ -303,6 +303,10 @@ type BuildStrategy struct {
>
>  	// CustomStrategy holds the parameters to the Custom build strategy
>  	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
> +
> +	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
> +	// This strategy is experimental.

How fast can we get an origin image with the plugin?

all depends what we're willing to live with. i can build and publish what
we have today if we want (though I do need to make sure introducing the
pipeline and sync plugins to our image don't break any of our existing
jenkins flows)

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/7949/files/fb7993e7a126f0d0a611f5d9aa5fad48b35d5e76#r61367900>